.changeset/itchy-items-serve.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs-backend': minor
```

Comment:
Assuming this is a big it would just be a `patch` change

Reply:
Coming right up

Reply:
bug* 😁